pkg/scannerv4/mappers/mappers.go

```diff
@@ -695,6 +765,47 @@ func pkgFixedBy(enrichments map[string][]json.RawMessage) (map[string]string, er
 	return pkgFixedBys, nil
 }
 
+// cveEPSS unmarshals and returns the EPSS enrichment, if it exists.
+func cveEPSS(ctx context.Context, enrichments map[string][]json.RawMessage) (map[string]epss.EPSSItem, error) {
+	enrichmentList := enrichments[epss.Type]
+	if len(enrichmentList) == 0 {
+		zlog.Warn(ctx).
+			Str("enrichments", epss.Type).
+			Msg("no EPSS enrichments found")
+		return nil, nil
+	}
+
+	var epssItems map[string][]epss.EPSSItem
+	err := json.Unmarshal(enrichmentList[0], &epssItems)
+	if err != nil {
+		return nil, fmt.Errorf("unmarshaling EPSS enrichment: %w", err)
+	}
+
+	if len(epssItems) == 0 {
+		return nil, nil
+	}
+
+	res := make(map[string]epss.EPSSItem)
```

Comment:
See reason for this suggestion in a comment below

```suggestion
	ret := make(map[string]map[string]epss.EPSSItem)
```